sdk/healthbot/Microsoft.Azure.Management.Healthbot/src/Microsoft.Azure.Management.Healthbot.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$(RepoEngPath)/mgmt/AzSdk.reference.props" />
+  <!-- Please do not move/edit code below this line -->
+
+  <PropertyGroup>
+    <Description> Provides developers with libraries for deploying and manageing Azure HealthBots. </Description>
+    <Version>1.0.0</Version>
```

Comment:
Is stable version intentional? If swagger is in preview or SDK is in preview, the version number recommended to start at 1.0.0-beta.1

Reply:
@allenjzhang The swagger is not in preview. Should the SDK be in preview if it's the first time it is released?

Reply:
Typically service go preview first then GA. But it is your group's choice.